addOns/openapi/src/main/resources/org/zaproxy/zap/extension/openapi/resources/Messages.properties

```diff
@@ -5,21 +5,32 @@ openapi.api.action.importUrl = Import an Open API definition from a URL, hostOve
 
 openapi.cmdline.file.help = Import an Open API definition from the specified file name
 openapi.cmdline.url.help = Import an Open API definition from the specified URL
+openapi.cmdline.targeturl.help = Use in conjunction with -openapifile, this option is necessary in case the definition has no servers section
+openapi.cmdline.siteoverride.help = Use in conjunction with -url, scan the specified host instead 
 
 openapi.desc = Allows you to spider and import OpenAPI (Swagger) definitions 
 openapi.topmenu.import.importopenapi = Import an Open API definition from the local file system
 openapi.topmenu.import.importopenapi.tooltip = The file must be a formal described OpenApi file.
 openapi.topmenu.import.importremoteopenapi = Import an Open API definition from a URL
 openapi.topmenu.import.importremoteopenapi.tooltip = The file must be a formal described OpenApi file.
 
-openapi.importfromurldialog.pasteaction = Paste
 openapi.importfromurldialog.labeloverride = Override Host:
 openapi.importfromurldialog.labelurl = URL Pointing to OpenAPI defn:
 openapi.importfromurldialog.importbutton = Import
 openapi.importfromurldialog.title = Import OpenAPI Definition from URL
 openapi.importfromurldialog.badurl = Invalid URL
 openapi.importfromurldialog.badoverride = Invalid Override Host: this should be of the form "host" or "host:port":\n{0}
 
+openapi.importfromfiledialog.labeltarget = Target URL:
```

Comment:
IMO "Target URL" is misleading, "Fallback URL" would describe better the behaviour. Although for consistency this should behave as "Override Host" (if it's specified it's used) and we could use the same command line argument for URL and file.

Reply:
The behavior can be changed to be similar to "Override Host" . 
But for the command line argument: the problem is that when a local file without scheme and host is imported , we need a full URL, whereas when the definition is imported from a URL, only host is needed.

Reply:
I was referring specifically to the behaviour ("if it's specified it's used"), not the actual values the argument would accept (or how it would be named for that matter). Yeah, the parameter would have to accept the whole URI for both cases (we want to allow to override/specify all the URI components).

Reply:
I was referring to the possibility of using the same command line argument for URL and file, which means that "Override Host" will become "Target URL" as well.This sounds good as it allows users to override scheme as well when importing from URL, but will break the old API, what do you think?

Reply:
We don't need to require the whole URL we can still accept just `host[:port]` as before (so it would be, e.g. `[scheme://]host[:port][/path]`?).

Reply:
But still the name of this parameter should not be  "Override Host", and if we try to keep the old parameter and add this parameter which accepts `[scheme://]host[:port][/path]` ,  this will complicate things even more.
So it seems to me that if we want to use the same parameter for both cases , importing from file and URL, the API and command line arguments will need to be changed. Otherwise , we can keep the behavior of importing from URL as it is, and add the new "Target URL" only for importing from file , but rename it or use it if specified, as described in the first comment

Reply:
The only "problem" is in the API endpoint `importUrl` which must keep the same name `hostOverride`, IMO that's addressed with documentation. In all other cases (GUI, command line and API `importFile`) we can use whatever/correct name we want ("Target URL" sounds good).

Reply:
750a28df6a55fd72064282a9034414ec81739bec should achieve  this